library/src/main/java/com/owncloud/android/lib/resources/files/CreateFolderRemoteOperation.java

```diff
@@ -46,13 +46,27 @@ public class CreateFolderRemoteOperation extends RemoteOperation<String> {
     public CreateFolderRemoteOperation(String remotePath, boolean createFullPath) {
         this.remotePath = remotePath;
         this.createFullPath = createFullPath;
+        this.sessionTimeOut = new SessionTimeOut(30000, 5000);
     }
 
     public CreateFolderRemoteOperation(String remotePath, boolean createFullPath, String token) {
         this(remotePath, createFullPath);
         this.token = token;
     }
 
+    public CreateFolderRemoteOperation(String remotePath, boolean createFullPath, SessionTimeOut sessionTimeOut) {
+        this.remotePath = remotePath;
```

Comment:
Is there a particular reason why we aren't just calling this constructor from the other ones with the default session timeouts?